src/app/app.component.html

```diff
@@ -23,7 +23,7 @@ <h2>Unsupported Browser</h2>
     [blogUrlPrefix]="blogUriPrefix"
     [experienceUrlPrefix]="experienceUriPrefix"
     [isLoggedIn]="isLoggedIn"
-    [myAccountFlagEnabled]="true"
```

Comment:
myAccountFlagEnabled is no longer needed to be passed, the component now treats it as always enabled. Flag was just there temporarily for dealing with my account as a new feature in the past.